examples/production/openshift-values.yaml

```diff
@@ -0,0 +1,120 @@
+global:
+  openshift: true
+  spire:
+    jwtIssuer: https://oidc-discovery.$APP_SUBDOMAIN
+  telemetry:
+    prometheus:
+      enabled: true
+
+spire-server:
+  tornjak:
+    image:
+      registry: ghcr.io
+      repository: spiffe/tornjak-backend
+      tag: ubi-v1.4.1
+    ingress:
+      enabled: true
+      annotations:
+        route.openshift.io/termination: "edge"
+      hosts:
+        - host: spire-tornjak-backend.$APP_SUBDOMAIN
+          paths:
+            - path: /
+              pathType: Prefix
+  nodeAttestor:
+    k8sPsat:
+      serviceAccountAllowList: ["spire-system:spire-agent"]
+  notifier:
+    k8sbundle:
+      namespace: spire-system
+  podSecurityContext:
+    runAsUser: null
+    runAsGroup: null
+    fsGroup: null
+
+  ingress:
+    enabled: true
+    annotations:
+      route.openshift.io/termination: "passthrough"
+    hosts:
+      - host: spire-server.$APP_SUBDOMAIN
+        paths:
+          - path: ''
+            pathType: ImplementationSpecific
+
+
+spire-agent:
+  podSecurityContext:
+    runAsUser: null
+    runAsGroup: null
+    fsGroup: null
+
+upstream-spire-agent:
+  podSecurityContext:
+    runAsUser: null
+    runAsGroup: null
+    fsGroup: null
+
+spiffe-oidc-discovery-provider:
+
+  podSecurityContext:
+    runAsUser: null
+    runAsGroup: null
+    fsGroup: null
+  securityContext:
+    allowPrivilegeEscalation: true
+    runAsNonRoot: false
+    privileged: true
+  ingress:
+    enabled: true
+    annotations:
+      route.openshift.io/termination: "edge"
+    hosts:
+      - host: oidc-discovery.$APP_SUBDOMAIN
+        paths:
+          - path: /
+            pathType: Prefix
+
+tornjak-frontend:
+  apiServerURL: https://spire-tornjak-backend.$APP_SUBDOMAIN/
+  workingDir: /opt/app-root/src
```

Comment:
Interesting.... why does this need to change? this a selinux thing? Should we be considering setting this to default everywhere?

Reply:
arbitrary UID support